templates/taxi/car_detail.html

```diff
@@ -1,12 +1,15 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <p><h4>Car: {{ car }}</h4></p>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
   <h1>Drivers</h1>
+  <p><a class="link" href="{% url 'taxi:car-update' car.id %}">Click here to update the car</a></p>
+  <p><a class="link" href="{% url 'taxi:car-delete' car.id %}">Click here to delete the car</a></p>
   <hr>
   <ul>
-  {% for driver in car.drivers.all %}
-    <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
-  {% endfor %}
+    {% for driver in car.drivers.all %}
+      <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
+    {% endfor %}
   </ul>
```

Comment:
It seems that the indentation in your HTML file is not consistent. According to the checklist, you should use 2 whitespaces for indentation in your `.html` files. This is a code style issue. Please, review and fix the indentation.